iib/workers/tasks/build.py

```diff
@@ -1426,8 +1475,31 @@ def _add_ocp_label_to_index(ocp_version, temp_dir, dockerfile_name):
     :param str dockerfile_name: the dockerfile name.
     """
     with open(os.path.join(temp_dir, dockerfile_name), 'a') as dockerfile:
-        label = f'LABEL com.redhat.index.delivery.version="{ocp_version}"'
+        label = f'LABEL {label_key}="{label_value}"'
         dockerfile.write(f'\n{label}\n')
         log.debug(
             'Added the following line to %s: %s', dockerfile_name, label,
         )
+
+
+def _validate_distribution_scope(resolved_distribution_scope, distribution_scope):
+    """
+    Validate distribution scope is allowed to be updated.
+
+    :param str resolved_distribution_scope: the distribution_scope that the index is for.
+    :param str distribution_scope: the distribution_scope that has been requested the index is for.
+    :return: the valid distribution_scope
+    :rtype: str
+    :raises IIBError: if the resolved_distribution_scope is of lesser scope then distribution_scope
```

Comment:
Typo and should use double back ticks around variable names in the RST docstring.
```suggestion
    :raises IIBError: if the ``resolved_distribution_scope`` is of lesser scope than ``distribution_scope``
```